tests/reconfig/results/v1.0.0.md

```diff
@@ -3,8 +3,10 @@
 <!-- TOC -->
```

Comment:
Per the pattern of our other tests, we had a 1.0.0 directory under `results/`, which this file can live in.